python/py-ndindex/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-ndindex
+version             1.7
+revision            0
+categories-append   devel
+license             MIT
+
+python.versions     37 38 39 310 311
+
+maintainers         nomaintainer
+
+description         A Python library for manipulating indices of ndarrays
+
+long_description    ${description}
+
+homepage            https://quansight-labs.github.io/ndindex/
+
+checksums           rmd160  3e3fc8e48bc295567e49b3a37190c89a9009dfca \
+                    sha256  bf9bd0b76eeada1c8275e04091f8291869ed2b373b7af48e56faf7579fd2efd2 \
+                    size    85371
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                        path:bin/cython:py${python.version}-cython
+    depends_test-append \
+                        port:py${python.version}-hypothesis \
+                        port:py${python.version}-numpy \
```

Comment:
NumPy is listed by upstream under "extras_require" with the comment "# NumPy is only required when using array indices"; do you think this is a commonly used thing by end-users? If so, I would probably recommend to add it to `depends_lib-append` so that it will be available at runtime.

Reply:
Thanks @reneeotten for your review. I followed your recommendation.